cmp/grid/Grid.js

```diff
@@ -245,7 +244,6 @@ class LocalModel {
             groupRowInnerRenderer: model.groupRowRenderer,
             groupRowRendererFramework: model.groupRowElementRenderer,
             groupRowRendererParams: {suppressCount: !model.showGroupRowCounts},
-            rememberGroupStateWhenNewData: true, // turning this on by default so group state is maintained when apps are not using immutableData
```

Comment:
Why did we remove this?

Reply:
See https://github.com/ag-grid/ag-grid/blob/3ed885a9cde6742d5c2469093f02b424d9c2bb05/community-modules/core/dist/es6/gridOptionsWrapper.js#L1123